contribute/create-gpg-keys.md

```diff
@@ -118,6 +99,7 @@ You should ask a PMC member to complete this step.
 svn co https://dist.apache.org/repos/dist/release/pulsar pulsar-dist-release-keys --depth empty
```

Comment:
```suggestion
svn co https://dist.apache.org/repos/dist/release/pulsar pulsar-dist-release-keys --depth files
```

I actually use files so the only file `KEYS` is downloaded properly. Maybe you can give it a try and see if it's a better/more concise approach.

Reply:
Yes, it works for me.